src/solCompile.ts

```diff
@@ -60,14 +78,16 @@ type SolcInput = {
   };
 };
 
-function formatInput(fileName: string): SolcInput {
+function formatInput(fileNames: string[]): SolcInput {
+  const sources: { [key: string]: { urls: string[] } } = {};
```

Comment:
minor nit: let's make this a reduce

Reply:
This will just make it longer since we'll still need to declare 'sources' to get the types to work. I already tried it.

Reply:
```
  const sources = fileNames.reduce((sources, fileName) => {
    sources[fileName] = {
      urls: [fileName],
    };
    return sources;
  }, {} as { [key: string]: { urls: string[] } });
  ```